### DISCUSSION TITLE
Error 402 when deformation is large

### DISCUSSION THREAD

**Discussion by 123fgh123** [damask-multiphysics/DAMASK#50] [Q&A]
Dear Colleagues,

Recently, I used Damask to conduct CP simulation under shear loading. 

I found that when the deformation is quite large (e.g. shear strain ~2), it would lead to a error with code 402. I do not know how to adjust my simulation setup to avoid this error. 

` ┌─────────────────────────────────────────────────────────────────────┐
 │                        error 402                                    │
 ├─────────────────────────────────────────────────────────────────────┤
 │ invalid orientation specified                                       │
 │ fromMatrix                                                          │
 │                                                                     │
 └─────────────────────────────────────────────────────────────────────┘`

I have attached my simulation files and logs for further check. Thank you!

[Error-402.zip](https://github.com/user-attachments/files/18135842/Error-402.zip)

Best regards.

**Comment by 123fgh123** [comment]
I want to simulate the material response under the shear strain ~2, but the simulation terminated in advance at shear strain ~1.

**Reply by eisenlohr** [reply]
Numerical convergence after substantial straining (order 1) is rather difficult. This is to be expected and might be circumvented by utilizing regridding.

**Reply by 123fgh123** [reply]
Thank you. Is there any tutorial about the setup of regridding in Damask ?

**Reply by 123fgh123** [reply]
I want to regrid the simulation results and restart it, and I have tried the fuction "damask.grid_filters.regrid()" and it works well. 

However, I found that the simulation restarted only based on the simulation results before regriding.

Is it possible  to restart the simulation based on the results after regriding?
